src/adcm_pytest_plugin/docker_utils.py

```diff
@@ -325,6 +329,32 @@ def __init__(self, container: Container, ip: str, port: str, container_config: O
     def stop(self):
         """Stops container"""
         self.container.stop()
+        with suppress(NotFound):
+            condition = "removed" if self.container.attrs["HostConfig"]["AutoRemove"] else "stopped"
+            self.container.wait(condition=condition, timeout=30)
+
+    @allure.step("Upgrade adcm to {target}")
+    def upgrade(self, target: Tuple[str, str]) -> None:
```

Comment:
This method should raise an error if no volume was mounted to the ADCM container before.

Reply:
Done.